kube-derive/Cargo.toml

```diff
@@ -19,6 +19,7 @@ proc-macro2.workspace = true
 quote.workspace = true
 syn = { workspace = true, features = ["extra-traits"] }
 serde_json.workspace = true
+k8s-openapi = { workspace = true, features = ["latest"] }
```

Comment:
this feels unfortunate. don't really want a derive crate (which is hard to optimise compile wise) to depend on such a big crate.